src/Builder.hs

```diff
@@ -103,6 +117,9 @@ data Builder = Alex
              | Tar TarMode
              | Unlit
              | Xelatex
+             | CabalFlags Stage -- ^ a virtual builder to use the Arg predicate logic
+                                --   to collect cabal flags. +x, -x
```

Comment:
What does `+x, -x` mean? Please clarify.

Reply:
They refer to cabal flags (+x meaning enable flag x, -x means disable flag x). Should I expand with such a wording instead?

Reply:
Yes, please. Perhaps, add an example.

Reply:
Fixed in a new commit.